src/com/palmergames/bukkit/towny/object/statusscreens/StatusScreen.java

```diff
@@ -7,48 +7,54 @@
 import java.util.List;
 import java.util.Map;
 
+import com.palmergames.bukkit.towny.utils.TownyComponents;
 import org.bukkit.command.CommandSender;
-import org.bukkit.util.ChatPaginator;
-
-import com.palmergames.bukkit.util.Colors;
-import com.palmergames.util.StringMgmt;
 
 import net.kyori.adventure.text.Component;
-import net.kyori.adventure.text.TextComponent;
 import net.kyori.adventure.text.event.ClickEvent;
 import net.kyori.adventure.text.event.HoverEvent;
+import org.bukkit.util.ChatPaginator;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
 
 public class StatusScreen {
 
-	Map<String, Component> components = new LinkedHashMap<>();
-	final static int MAX_WIDTH = ChatPaginator.AVERAGE_CHAT_PAGE_WIDTH;
-	private CommandSender sender;
+	private static final int MAX_WIDTH = 320;
+
+	private final Map<String, Component> components = new LinkedHashMap<>();
+	private final CommandSender sender;
 	
 	public StatusScreen(CommandSender sender) {
 		this.sender = sender;
 	}
 
+	@Deprecated
+	@ApiStatus.ScheduledForRemoval
 	public CommandSender getCommandSender() {
 		return sender;
 	}
 
+	@Deprecated
 	public void addComponentOf(String name, String text) {
-		components.put(name, Component.text(text));
+		components.put(name, TownyComponents.miniMessage(text));
 	}
 
 	public void addComponentOf(String name, Component component) {
 		components.put(name, component);
 	}
 	
+	@Deprecated
 	public void addComponentOf(String name, String text, ClickEvent click) {
 		components.put(name, Component.text(text).clickEvent(click));
 	}
 	
-	public void addComponentOf(String name, String text, HoverEvent hover) {
+	@Deprecated
+	public void addComponentOf(String name, String text, HoverEvent<?> hover) {
 		components.put(name, Component.text(text).hoverEvent(hover));
 	}
 	
-	public void addComponentOf(String name, String text, HoverEvent hover, ClickEvent click) {
+	@Deprecated

```

Comment:
Needs the `@Deprecated since....` should probably also explain what to replace it with.